readme.md

```diff
@@ -43,7 +44,7 @@ Elements in beautiful READMEs include, but are not limited to: images, screensho
 - [supunlakmal/thismypc](https://github.com/supunlakmal/thismypc#readme) - Project logo. Useful badges. Web site URL. Screenshot. GIF step-by-step instructions for installation. Project folder structure.
 - [teles/array-mixer](https://github.com/teles/array-mixer#readme) - Clean project logo, badges. Table of contents. Contributing section. Many illustrated examples.
 - [thelounge/thelounge](https://github.com/thelounge/thelounge#readme) - Project logo. Useful badges and links (website, docs, demo). Screenshot. Feature list. Step-by-step instructions for installation, development, and testing. Link to the contributing guide. Good mobile summary.
-- [vhesener/Closures](https://github.com/vhesener/Closures#readme) - Project logo, cognitive funnel, animated examples. Color coordinated. Clean documentation.
+- [vhesener/Closures](https://github.com/GideonWolfe/vim.reaper) - Project logo, cognitive funnel, animated examples. Color coordinated. Clean documentation.
```

Comment:
This change looks like it was done in error?

Reply:
Anything else need fixing?